### ISSUE TITLE
Create CLI

### ISSUE THREAD

**Issue by kleberapsilva** [ShadyBoukhary/flutter_clean_architecture#35]
Could we create a CLI like Angular to facilitate the creation of files?

**Comment by ShadyBoukhary** [comment]
@kleberapsilva yes, that's currently in my agenda. Stay on the lookout for PRs.

**Comment by WilliamCunhaCardoso** [comment]
> Could we create a CLI like Angular to facilitate the creation of files?

For now, you can use this option: https://github.com/pbissonho/fast.cli

**Comment by kleberapsilva** [comment]
> > Could we create a CLI like Angular to facilitate the creation of files?
> 
> For now, you can use this option: https://github.com/pbissonho/fast.cli

Você já chegou a configurar os templates utilizando o clean?

**Comment by WilliamCunhaCardoso** [comment]
> > > Could we create a CLI like Angular to facilitate the creation of files?
> > 
> > 
> > For now, you can use this option: https://github.com/pbissonho/fast.cli
> 
> Você já chegou a configurar os templates utilizando o clean?

No momento, não.

No projeto que usamos este pacote utilizamos scripts bash.

Em outros que são meus utilizo o slidy do modular.

Citei esse porque conheci na comunidade Flutterando e acredito ser viável para essa proposição.

**Comment by ShadyBoukhary** [comment]
I'm going to implement a CLI very soon.

**Comment by raphaaugustosilva** [comment]
Do you have an update about the CLI development? Thanks!

**Comment by ShadyBoukhary** [comment]
I'm only very rarely working on this library anymore since i'm working full time and @rafaelcmm 's team have been maintaining this library for the most part. It's unlikely that i'll have time to make a CLI as it's only a minor improvement. Anyone willing to build that should feel free to do so. It will be appreciated. 

Shady
